prymer/primer3/primer3_parameters.py

```diff
@@ -186,6 +188,14 @@ class ProbeParameters:
 
     Please see the Primer3 manual for additional details: https://primer3.org/manual.html#globalTags
 
+    Primer3 uses both thermodynamic and alignment-based approaches to quantify oligo
+    self-complementarity. In general, these settings are meant to limit problematic oligo
+    self-complementarity and avoid primer-dimers or other nonspecific binding of probes.
+
+    If they are not provided, `probe_max_self_any_thermo`, `probe_max_self_end_thermo`, and
+    `probe_max_hairpin_thermo` will be set to default values as specified in the Primer3 manual.
+    The default value is 10 degrees lower than the minimal melting temperature specified for
+    probe design.
```

Comment:
Can we have both deltaG and Tm? I've been using deltaG as a filter.

The docstring here should be very clear that if a user sets any of these attributes, they have to be *absolute* values, even though the default is relative to the min probe Tm.

Reply:
thanks Alison! I've updated the docstring in the most recent commits 👍